README.md

```diff
@@ -1,94 +1,87 @@
 
-The framework provides a testing harness, so you can ensure that your charm
-does the right thing in different scenarios, without having to create
-a full deployment. Our [API documentation](https://ops.readthedocs.io/en/latest/#module-ops.testing)
-has the details, including this example:
+# Use the name from the packing step and the resource from the ‘metadata.yaml’
+# to deploy your example charm to your local MicroK8s cloud, for example:
+juju deploy ./my-new-charm_ubuntu-22.04-amd64.charm --resource httpbin-image=kennethreitz/httpbin
 
-```python
-class TestCharm(unittest.TestCase):
-    def test_foo(self):
-        harness = Harness(MyCharm)
-        self.addCleanup(harness.cleanup)  # always clean up after ourselves
+# Congratulations, you’ve just built your first charm!
 
-        # Instantiate the charm and trigger events that Juju would on startup
-        harness.begin_with_initial_hooks()
+# Now you can go back and edit the files to make the charm your own.
 
-        # Update charm config and trigger config-changed
-        harness.update_config({'log_level': 'warn'})
+# Once you’re done, come back to Charmcraft to publish and release your charm on Charmhub!
 
-        # Check that charm properly handled config-changed, for example,
-        # the charm added the correct Pebble layer
-        plan = harness.get_container_pebble_plan('prometheus')
-        self.assertIn('--log.level=warn', plan.services['prometheus'].command)
+# Clean up:
+# - delete the vm:
+multipass delete --purge my-vm
+# - uninstall multipass:
+snap remove multipass
 ```
+For more, see our user docs: https://juju.is/docs/sdk 
 
+## Contribute to Ops
 
-## Talk to us
-
-If you need help, have ideas, or would just like to chat with us, reach out on
-the Charmhub [Mattermost].
-
-We also pay attention to the Charmhub [Discourse].
-
-And of course you can deep dive into the [API reference].
-
-[Discourse]: https://discourse.charmhub.io/
-[API reference]: https://ops.readthedocs.io/
-[Mattermost]: https://chat.charmhub.io/charmhub/channels/charm-dev
-
+Read our [Code of conduct](https://ubuntu.com/community/code-of-conduct) and start chatting with us:
+- Join our chat: [Mattermost](https://chat.charmhub.io/charmhub/channels/charm-dev)
+- Join our forum: [Discourse](https://discourse.charmhub.io/)
+ 
+File an issue:
+- Report an Ops bug on [GitHub](https://github.com/canonical/charmcraft/issues)
+- Raise a general https://juju.is/docs documentation issue on [GitHub | juju/docs](https://github.com/juju/docs)
 
-## Development of the framework
+Make a change:
+- Read our [documentation contributor guidelines](https://discourse.charmhub.io/t/documentation-guidelines-for-contributors/1245) and help improve a doc
+- Read our [codebase contributor guidelines](https://github.com/canonical/operator/blob/main/HACKING.md) and our [API reference](https://ops.readthedocs.io/en/latest/) and help improve the codebase
```

Comment:
```suggestion
- Read our [codebase contributor guidelines](HACKING.md) and our [API reference](https://ops.readthedocs.io/en/latest/) and help improve the codebase
```

Reply:
I feel like this hides away the API reference docs link a bit - it feels like it's implying that it's for contributors, but it's really for users. I think if anything it should be _more_ prominent than in the current version rather than _less_.

(It seems like a properly Canonical imprrovement would be to have right at the top the quadrant of docs).

Reply:
I agree the API reference is user material and belongs in the user docs. I'll move it there. (It's already implicit at the end of the Use section, in the link the Charm SDK docs, but I can try to highlight it more directly by adding also an explicit link to the Charm SDK doc on Ops.